rssfeed/models.py

```diff
@@ -4,10 +4,11 @@
 class Feed(models.Model):
     title = models.CharField(max_length=2000, blank=True, null=True)
     xml_url = models.CharField(max_length=255, unique=True)
```

Comment:
Just call it URL. No need to imply the format in the field name, plus you limit yourself if a new non-XML RSS format pops up.